server/src/infra/entities/asset.entity.ts

```diff
@@ -82,6 +82,9 @@ export class AssetEntity {
   @Index()
   checksum!: Buffer; // sha1 checksum
 
+  @Column()
+  projectionType!: ProjectionType;
```

Comment:
We can remove it since `ExifEntity` already has this value

Reply:
Moved to exif